test/functional/services/combo_box.js

```diff
@@ -36,7 +36,14 @@ export function ComboBoxProvider({ getService }) {
       log.debug(`comboBox.setElement, value: ${value}`);
       await this._filterOptionsList(comboBoxElement, value);
       await this.openOptionsList(comboBoxElement);
-      await find.clickByCssSelector('.euiComboBoxOption');
+
+      // if it doesn't find the item which text starts with value, it will choose the first option
+      try {
+        await find.clickByCssSelector(`.euiComboBoxOption[title^="${value.trim()}"]`);
```

Comment:
@dmlemeshko could you please review the changes in the functional tests?

Reply:
@dmlemeshko I updated the test including the [feedback](https://github.com/elastic/kibana/pull/33571#discussion_r268351570) from @lukeelmers in #33571 (I hope that this PR will be merge earlier than #33571 and I will rebase the changes to #33571 ). 

I run locally, the tests passed. The CI is green as well. Could you please review again?